core/database_api.php

```diff
@@ -451,32 +451,43 @@ function db_result( $p_result, $p_row_index = 0, $p_col_index = 0 ) {
 }
 
 /**
- * return the last inserted id for a specific database table
+ * Return the last inserted ID after a insert statement.
+ * Warning: this function must be used immediately after the insert statement
+ *
+ * This relies in AdoDB to get the entity id when this functionality is available
```

Comment:
on ADOdb

Reply:
This wasn't fixed... `in AdoDB` -> `on ADOdb`

Reply:
Sorry for picking nits, it's *rely **on***, not *in*
https://dictionary.cambridge.org/dictionary/english/rely-on-upon-sb-sth

Reply:
> _**on**_, not _in_

sorry i din't notice that

the corrections are welcome, thanks